app/helpers/application_helper.rb

```diff
@@ -276,4 +276,23 @@ def sign_in_path
   def unread_notifications(user)
     Comment.accessible_by(current_ability).find_since_last_login(user)
   end
+
+  def date_s(datetime)
```

Comment:
In two months I'm going to have to look up what `date_s` is. I don't have to look up what `strftime` is, I know that. Please do not obfuscate...